### DISCUSSION TITLE
Bug Report: html5 doesn't work

### DISCUSSION THREAD

**Discussion by itsaban** [KadeArchive/Kade-Engine#530] [Q&A]
so yeah i tried to compile html5 but after compile this don't working.

i tried start it with localhost, upload to the server... but it's not working at all

terminal output after compile:
C:/HaxeToolkit/haxe/lib/linc_luajit/0,0,2/llua/Lua.hx:492: characters 43-61 : Abstract<cpp.Callable> has no field fromStaticFunction
C:/HaxeToolkit/haxe/lib/linc_luajit/0,0,2/llua/Lua.hx:526: characters 42-60 : Abstract<cpp.Callable> has no field fromStaticFunction
C:/HaxeToolkit/haxe/lib/polymod/git/polymod/format/XMLMerge.hx:189: characters 10-23 : Warning : This typedef is deprecated in favor of haxe.xml.Access
C:/HaxeToolkit/haxe/lib/polymod/git/polymod/format/XMLMerge.hx:189: characters 30-43 : Warning : This typedef is deprecated in favor of haxe.xml.Access

here some output from browser's console:
Failed to load resource: net::ERR_FILE_NOT_FOUND
index.html:75 Uncaught ReferenceError: lime is not defined
    at index.html:75
    

please help me

**Comment by ghost** [comment]
I got this with another mod framework but not with KE.
I have no idea why it does this.

**Comment by suprstarrd** [comment]
are you sure you have `lime` installed?

**Reply by ghost** [reply]
I got this with another framework, do have Lime installed

**Reply by suprstarrd** [reply]
huh, interesting

**Reply by ghost** [reply]
But I can compile KE fine for HTML5

**Reply by itsaban** [reply]
yes, lime installed

**Reply by suprstarrd** [reply]
how did you download the repository?
